managed/src/main/java/com/yugabyte/yw/controllers/SessionController.java

```diff
@@ -362,7 +362,7 @@ public Result appVersion() {
   @With(TokenAuthenticator.class)
   public Result getLogs(Integer maxLines) {
     String appHomeDir = appConfig.getString("application.home", ".");
-    String logDir = System.getProperty("log.override.path", String.format("%s/logs", appHomeDir));
+    String logDir = appConfig.getString("log.override.path", String.format("%s/logs", appHomeDir));
```

Comment:
so I got a hit for `log.override.path` in bin/yugabyted so we should keep the prev behavior.

I would suggest doing this in a fall back fashion - first use the property if defined, then look in the app config, then default to home dir.

Reply:
Yes, that was the only place which I found to be using `-Dlog.override.path`. This change still works with that way as well. To confirm that, I tried to set different path in application.conf and started the platform with `-Dlog.…`, it took the value which is given on command line.